web/packages/teleterm/src/ui/TopBar/Connections/ConnectionsFilterableList/ConnectionStatusIndicator.tsx

```diff
@@ -27,33 +26,46 @@ export const ConnectionStatusIndicator = (props: {
   [key: string]: any;
 }) => {
   const { status, ...styles } = props;
-  return <StyledStatus $status={status} {...styles} />;
+
+  if (status === 'error') {
+    // Using text instead of an icon because any icon used here would be smaller than the rounded
```

Comment:
So the used icon is so small on purpose?
To me it is too tiny compared to other state icons, I tested how it looks with font size 20px:
<img width="343" alt="image" src="https://github.com/gravitational/teleport/assets/20583051/70897899-fb85-4f7c-b472-50e32bfc6bc7">

Personally, I like it better :)

Reply:
It indeed looks better when it comes to size, but it's not aligned vertically with other states. Thoughts on the abomination that I just pushed?

| New | Overlaid on other states |
| --- | --- |
| <img width="356" alt="new" src="https://github.com/gravitational/teleport/assets/27113/f23b7b0e-4139-4850-b564-5cbb0e5251f1"> | <img width="356" alt="overlaid" src="https://github.com/gravitational/teleport/assets/27113/a674a5b4-6589-4a01-bee3-b2c107d28720"> |

Reply:
It's not beautiful, but I think we often do it for icons. I'm okay with it.